draft-ietf-tls-dtls13.md

```diff
@@ -1174,69 +1174,88 @@ is received. (See {{dtls-epoch}} for the definitions of each epoch.)
 
 ##  DTLS Handshake Flights
 
-DTLS messages are grouped into a series of message flights, according
-to the diagrams below.
+DTLS handshake messages are grouped into a series of message flights. A flight starts with the 
+handshake message transmission of one peer and ends with the expected response from the 
+other peer. Table {{tab-flights}} contains a complete list of message combinations that consitute flights. 
+
+| Note | Client | Server | Handshake Messages |
+|-----+--------+--------+------------------ |
+|     |    x   |        | ClientHello |
+| (1) |        |    x   | HelloRetryRequest |
+|     |        |    x   | ServerHello, EncryptedExtensions, CertificateRequest*, Certificate*, CertificateVerify*, Finished | 
+| (2) |    x   |        | Certificate*, CertificateVerify*, Finished |
+|     |        |    x   | ACK | 
+| (3) |        |    x   | NewSessionTicket | 
+{: #tab-flights title="Flight Handshake Message Combinations."}
+
+Remarks: 
+
+- The HelloRetryRequest shown in (1) is an optional message sent by the server. 
+- The ACK message is sent by a client in response to a NewSessionTicket message from the server (as noted in (3)) or sent by a server in response to receipt of flight (2). In those cases it represent a flight on its own. 
```

Comment:
I tend to think we should not think of an ACK as part of a flight. It's kind of outside this model.